develop-docs/sdk/telemetry/spans/span-protocol.mdx

```diff
@@ -148,7 +148,7 @@ The envelope item payload must contain an `items` array with span one and up to
 | `parent_span_id` | string | No | 16-character hexadecimal string (a valid uuid4 without dashes) |
 | `name` | string | Yes | A low cardinality description of what the span represents (e.g., "GET /users", "database.query") |
 | `status` | string | Yes | Status of the span operation. Either `"ok"` or `"error"` |
-| `is_remote` | boolean | Yes | Whether the SpanContext creating the span was received from somewhere else or locally generated |
+| `is_segment` | boolean | Yes | Whether the span is a segment span |
```

Comment:
Technically we decided not to require it for compatibility reasons with otel, but I am good with keeping it as required here for our own SDKs 👍.